### ISSUE TITLE
MSync is not saving users' ranks to the mrSync table

### ISSUE THREAD

**Issue by Bryantdl7** [Aperture-Development/MSync-2#31]
I was able to get everything connected last night after my rookie mistake. Database tables are up, server is populated, however, nothing else has copied over. 

I have specified ranks which will synchronize to the all servers category, enabled the module, chose sync, rebooted, tried syncing again, and left the server running at idle all night. 

All I was able to get was on disconnect, an entry for my player in the following table: 

![image](https://user-images.githubusercontent.com/5564661/81712115-41264180-9442-11ea-80d5-138dc58f3427.png)

mrsync table:
![image](https://user-images.githubusercontent.com/5564661/81713183-8b5bf280-9443-11ea-9534-570aefed8d5d.png)

My settings.txt - cleaned confidential details out: 

```
{

        "mysql": {
                "host": "localhost",
                "database": "msync",
                "username": "msync",
                "password": "1-1-1-1-1-1-1-1-",
                "port": "3306"
        },
        "serverGroup": "darkstorm-all",
        "enabledModules": {
                "MRSync": true,
                "mrsync": true,
                "MBSync": true
        }
}

```

mrsync.txt 

```

{
        "nosync": [],
        "syncall": {
                "owner": true,
                "member": true,
                "retired-staff": true,
                "serveradmin": true,
                "grandmaster": true,
                "mod": true
        }
}




```

**Comment by ApertureDevelopment** [comment]
As I replied on Discord, MSync does not import data from ULX. MSync starts to record and save data from the point on where you added it to your server. When a user joins your server, a table entry for the user gets created, when he leaves the server MRSync saves the rank to the database, and when you add a ban using the !mban commands a ban gets added to the database.

**Comment by Bryantdl7** [comment]
I worded this incorrectly. I meant that it does not save users at all

**Comment by ApertureDevelopment** [comment]
Is a entry called "allservers" in the ``tbl_server_grp`` table?

**Comment by Bryantdl7** [comment]
As I was presented the option to rename this, I changed it to the following ingame: 

![image](https://user-images.githubusercontent.com/5564661/81713598-14732980-9444-11ea-9222-836f8e7998fe.png)

**Comment by Bryantdl7** [comment]
Upon changing the value back to allservers this does indeed work. Possibly your logic is currently not set up to handle other group names.

**Comment by ApertureDevelopment** [comment]
There is the issue, the "allserver" group should be there by default, it is used to identify ranks that are synchronized across all servers. 

create the entry using this:
``INSERT INTO `tbl_server_grp` (group_name) VALUES ('allservers');``

**Comment by ApertureDevelopment** [comment]
I will add a bugfix for that later, I btw also suggest to use our workshop addon to recieve automatic updates

https://steamcommunity.com/sharedfiles/filedetails/?id=1636008512

**Comment by Bryantdl7** [comment]
I plan to use the workshop addon in the future, I currently am running this on our development server in order to check for bugs before we put it into production

**Comment by ApertureDevelopment** [comment]
Fixed in v1.0.3
